packages/web-runtime/src/store/modal.ts

```diff
@@ -71,6 +71,7 @@ const mutations = {
     state.onConfirm = modal.onConfirm
     state.hasInput = modal.hasInput || false
     state.inputValue = modal.inputValue || null
+    state.inputPasswordPolicy = modal.inputPasswordPolicy || {}
```

Comment:
In `OcModal`, `inputPasswordPolicy` defaults to `[]`, not `{}`. Should this be aligned?

Reply:
Yes, this is outdated